apollo-router/src/plugins/telemetry/config_new/router/events.rs

```diff
@@ -21,9 +21,9 @@ use crate::services::router;
 
 #[derive(Clone)]
 pub(crate) struct DisplayRouterRequest(pub(crate) EventLevel);
-#[derive(Default, Clone)]
+#[derive(Default, Clone, Debug)]
 pub(crate) struct DisplayRouterResponse;
-#[derive(Default, Clone)]
+#[derive(Default, Clone, Debug)]
 pub(crate) struct RouterResponseBodyExtensionType(pub(crate) String);
```

Comment:
I don't think it should be part of this file anymore as it's not only for events but also for selector